releasenotes/notes/override-l2pop-default-d7426c07d8b72f2e.yaml

```diff
@@ -1,10 +1,4 @@
 ---
-issues:
-  - The default value for `neutron_l2_population` has been
-    overridden, which will require deployers to recreate
-    any VXLAN interfaces created with `neutron_l2_population`
-    set `False`. Not recreating these interfaces will lead to
-    connectivity issues for virtual machines.
-upgrade:
+other:
   - Based on feedback from Support, the default value for
```

Comment:
``Based on feedback from our support team, ``

It sounds better if it's to be public.